String length in warning message violating ISO C90 compiler requirements

Simulating models that use `IBPSA.Fluid.Movers.BaseClasses.PartialFlowMachine` yields the following warning with OCT (v 1.40).
```
sources/IBPSA_Fluid_Movers_Validation_SpeedControlled_y_equ_init.c:740:9: warning: string length '663' is greater than the length '509' ISO C90 compilers are required to support [-Woverlength-strings]
         jmi_assert_failed("*** Warning: You are using a flow or pressure controlled mover with the\n             default pressure curve with per.etaHydMet set to\n             IBPSA.Fluid.Movers.BaseClasses.Types.HydraulicEfficiencyMethod.Power_VolumeFlowRate.\n             Since this can cause wrong power consumption, the model will overwrite\n
 this setting and use instead\n             IBPSA.Fluid.Movers.BaseClasses.Types.HydraulicEfficiencyMethod.NotProvided.See the user guide to see what assumptions are made in this setting\n             or provide other information to the model.\n             Setting nominalValuesDefineDefaultPressureCurve=true will suppress this warning.", JMI_ASSERT_WARNING);
```
Note that this occurs even if the assert condition is true.

--- a/IBPSA/Fluid/Movers/BaseClasses/PartialFlowMachine.mo
+++ b/IBPSA/Fluid/Movers/BaseClasses/PartialFlowMachine.mo
@@ -395,12 +395,10 @@ initial algorithm
   assert(nominalValuesDefineDefaultPressureCurve or
          per.havePressureCurve or
          (preVar == IBPSA.Fluid.Movers.BaseClasses.Types.PrescribedVariable.Speed),
-"*** Warning: You are using a flow or pressure controlled mover with the
-             default pressure curve.
-             This leads to approximate calculations of the electrical power
-             consumption. Add the correct pressure curve in the record per
-             to obtain an accurate computation.
-             Setting nominalValuesDefineDefaultPressureCurve=true will suppress this warning.",
+"*** Warningin "+ getInstanceName()+
+": You are using a flow or pressure controlled mover with the default pressure curve.
+This leads to approximate calculations of the electrical power consumption.
+Setting nominalValuesDefineDefaultPressureCurve=true will suppress this warning.",
          level=AssertionLevel.warning);
 
   // The control signal is dp or m_flow but the user did not provide a fan or pump curve.
@@ -429,10 +427,10 @@ Setting nominalValuesDefineDefaultPressureCurve=true will suppress this warning.
             or per.etaHydMet ==
                IBPSA.Fluid.Movers.BaseClasses.Types.HydraulicEfficiencyMethod.EulerNumber),
 "*** Warning in "+ getInstanceName()+
-             ": per.etaHydMet is set to .Power_VolumeFlowRate or .EulerNumber.
-             This requires that per.pressure be provided.
-             Because it is not,
-             the model will override this setting and use .NotProvided instead.",
+": per.etaHydMet is set to .Power_VolumeFlowRate or .EulerNumber.
+This requires that per.pressure be provided.
+Because it is not,
+the model will override this setting and use .NotProvided instead.",
          level=AssertionLevel.warning);
 
   assert(per.havePressureCurve or per.haveWMot_nominal or
@@ -441,11 +439,11 @@ Setting nominalValuesDefineDefaultPressureCurve=true will suppress this warning.
             or per.etaMotMet ==
                IBPSA.Fluid.Movers.BaseClasses.Types.MotorEfficiencyMethod.GenericCurve),
 "*** Warning in "+ getInstanceName()+
-             ": per.etaMotMet is set to .Efficiency_MotorPartLoadRatio
-             or .GenericCurve. This requires that per.WMot_nominal be provided
-             or at least per.pressure be provided so that per.WMot_nominal can
-             be estimated. Because neither is provided,
-             the model will override this setting and use .NotProvided instead.",
+": per.etaMotMet is set to .Efficiency_MotorPartLoadRatio
+or .GenericCurve. This requires that per.WMot_nominal be provided
+or at least per.pressure be provided so that per.WMot_nominal can
+be estimated. Because neither is provided,
+the model will override this setting and use .NotProvided instead.",
          level=AssertionLevel.warning);
 
   assert(per.powerOrEfficiencyIsHydraulic or
@@ -454,11 +452,11 @@ Setting nominalValuesDefineDefaultPressureCurve=true will suppress this warning.
             or per.etaMotMet ==
                IBPSA.Fluid.Movers.BaseClasses.Types.MotorEfficiencyMethod.GenericCurve),
 "*** Warning in "+ getInstanceName()+
-             ": per.etaMotMet is set to .Efficiency_MotorPartLoadRatio
-             or .GenericCurve while information is provided for total electric
-             power instead of hydraulic power. This forms an algebraic loop
-             and may cause the simulation to not converge
-             (see the \"Motor efficiency\" section in usersguide).",
+": per.etaMotMet is set to .Efficiency_MotorPartLoadRatio
+or .GenericCurve while information is provided for total electric
+power instead of hydraulic power. This forms an algebraic loop
+and the simulation may fail to converge
+(see the \"Motor efficiency\" section in usersguide).",
          level=AssertionLevel.warning);
 
 equation
@@ -680,7 +678,7 @@ Added an <code>assert()</code> warning when the model has to make an unreliable
 guess for efficiency computation using <code>.EulerNumber</code>.
 </li>
 <li>
-Added an <code>assert()</code> warning when the model has to overwrite
+Added an <code>assert()</code> warning when the model has to override
 <code>per.etaMotMet</code>.
 </li>
 </ul>

--- a/IBPSA/Fluid/Movers/BaseClasses/PartialFlowMachine.mo
+++ b/IBPSA/Fluid/Movers/BaseClasses/PartialFlowMachine.mo
@@ -415,15 +415,12 @@ initial algorithm
            (preVar == IBPSA.Fluid.Movers.BaseClasses.Types.PrescribedVariable.Speed)) or
          per.etaHydMet<>
       IBPSA.Fluid.Movers.BaseClasses.Types.HydraulicEfficiencyMethod.Power_VolumeFlowRate,
-"*** Warning: You are using a flow or pressure controlled mover with the
-             default pressure curve with per.etaHydMet set to
-             IBPSA.Fluid.Movers.BaseClasses.Types.HydraulicEfficiencyMethod.Power_VolumeFlowRate.
-             Since this can cause wrong power consumption, the model will overwrite
-             this setting and use instead
-             IBPSA.Fluid.Movers.BaseClasses.Types.HydraulicEfficiencyMethod.NotProvided." +
-             "See the user guide to see what assumptions are made in this setting
-             or provide other information to the model.
-             Setting nominalValuesDefineDefaultPressureCurve=true will suppress this warning.",
+"*** Warning in "+ getInstanceName()+
+": You are using a flow or pressure controlled mover with the
+default pressure curve while per.etaHydMet is set to .Power_VolumeFlowRate.
+Since this can cause wrong power consumption, the model will override
+this setting and use instead .NotProvided.
+Setting nominalValuesDefineDefaultPressureCurve=true will suppress this warning.",
          level=AssertionLevel.warning);
 
   assert(per.havePressureCurve or

--- a/IBPSA/Fluid/Movers/BaseClasses/PartialFlowMachine.mo
+++ b/IBPSA/Fluid/Movers/BaseClasses/PartialFlowMachine.mo
@@ -395,10 +395,10 @@ initial algorithm
   assert(nominalValuesDefineDefaultPressureCurve or
          per.havePressureCurve or
          (preVar == IBPSA.Fluid.Movers.BaseClasses.Types.PrescribedVariable.Speed),
-"*** Warningin "+ getInstanceName()+
-": You are using a flow or pressure controlled mover with the default pressure curve.
-This leads to approximate calculations of the electrical power consumption.
-Setting nominalValuesDefineDefaultPressureCurve=true will suppress this warning.",
+"*** Warning in " + getInstanceName() +
+": Mover is flow or pressure controlled and uses default pressure curve.
+This leads to an approximate power consumption.
+Set nominalValuesDefineDefaultPressureCurve=true to suppress this warning.",
          level=AssertionLevel.warning);
 
   // The control signal is dp or m_flow but the user did not provide a fan or pump curve.
@@ -413,12 +413,11 @@ Setting nominalValuesDefineDefaultPressureCurve=true will suppress this warning.
            (preVar == IBPSA.Fluid.Movers.BaseClasses.Types.PrescribedVariable.Speed)) or
          per.etaHydMet<>
       IBPSA.Fluid.Movers.BaseClasses.Types.HydraulicEfficiencyMethod.Power_VolumeFlowRate,
-"*** Warning in "+ getInstanceName()+
-": You are using a flow or pressure controlled mover with the
-default pressure curve while per.etaHydMet is set to .Power_VolumeFlowRate.
-Since this can cause wrong power consumption, the model will override
-this setting and use instead .NotProvided.
-Setting nominalValuesDefineDefaultPressureCurve=true will suppress this warning.",
+"*** Warning in " + getInstanceName() +
+": Mover is flow or pressure controlled, uses default pressure curve and
+has per.etaHydMet=.Power_VolumeFlowRate.
+As this can cause wrong power consumption, the model overrides this setting by using per.etaHydMet=.NotProvided.
+Set nominalValuesDefineDefaultPressureCurve=true to suppress this warning.",
          level=AssertionLevel.warning);
 
   assert(per.havePressureCurve or
@@ -426,11 +425,10 @@ Setting nominalValuesDefineDefaultPressureCurve=true will suppress this warning.
                IBPSA.Fluid.Movers.BaseClasses.Types.HydraulicEfficiencyMethod.Power_VolumeFlowRate
             or per.etaHydMet ==
                IBPSA.Fluid.Movers.BaseClasses.Types.HydraulicEfficiencyMethod.EulerNumber),
-"*** Warning in "+ getInstanceName()+
-": per.etaHydMet is set to .Power_VolumeFlowRate or .EulerNumber.
-This requires that per.pressure be provided.
-Because it is not,
-the model will override this setting and use .NotProvided instead.",
+"*** Warning in " + getInstanceName() +
+": Mover has per.etaHydMet=.Power_VolumeFlowRate or per.etaHydMet=.EulerNumber.
+This requires per.pressure to be provided.
+Because it is not, the model overrides this setting by using per.etaHydMet=.NotProvided.",
          level=AssertionLevel.warning);
 
   assert(per.havePressureCurve or per.haveWMot_nominal or
@@ -438,12 +436,10 @@ the model will override this setting and use .NotProvided instead.",
                IBPSA.Fluid.Movers.BaseClasses.Types.MotorEfficiencyMethod.Efficiency_MotorPartLoadRatio
             or per.etaMotMet ==
                IBPSA.Fluid.Movers.BaseClasses.Types.MotorEfficiencyMethod.GenericCurve),
-"*** Warning in "+ getInstanceName()+
-": per.etaMotMet is set to .Efficiency_MotorPartLoadRatio
-or .GenericCurve. This requires that per.WMot_nominal be provided
-or at least per.pressure be provided so that per.WMot_nominal can
-be estimated. Because neither is provided,
-the model will override this setting and use .NotProvided instead.",
+"*** Warning in " + getInstanceName() +
+": Mover has per.etaMotMet=.Efficiency_MotorPartLoadRatio or per.etaMotMet=.GenericCurve.
+This requires per.WMot_nominal or per.pressure to be provided. Because neither is provided,
+the model overrides this setting and by using per.etaMotMet=.NotProvided.",
          level=AssertionLevel.warning);
 
   assert(per.powerOrEfficiencyIsHydraulic or
@@ -451,12 +447,11 @@ the model will override this setting and use .NotProvided instead.",
                IBPSA.Fluid.Movers.BaseClasses.Types.MotorEfficiencyMethod.Efficiency_MotorPartLoadRatio
             or per.etaMotMet ==
                IBPSA.Fluid.Movers.BaseClasses.Types.MotorEfficiencyMethod.GenericCurve),
-"*** Warning in "+ getInstanceName()+
-": per.etaMotMet is set to .Efficiency_MotorPartLoadRatio
-or .GenericCurve while information is provided for total electric
-power instead of hydraulic power. This forms an algebraic loop
-and the simulation may fail to converge
-(see the \"Motor efficiency\" section in usersguide).",
+"*** Warning in " + getInstanceName() +
+": Mover has per.etaMotMet=.Efficiency_MotorPartLoadRatio or per.etaMotMet=.GenericCurve
+and provides information for total electric power instead of hydraulic power.
+This forms an algebraic loop. If simulation fails to converge,
+see the \"Motor efficiency\" section in the users guide for how to correct it.",
          level=AssertionLevel.warning);
 
 equation

